app/src/main/java/org/mozilla/fenix/compose/CollectionsPlaceholder.kt

```diff
@@ -0,0 +1,164 @@
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.ext.dashedBorder
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+/**
+ * [CollectionsPlaceholder] for displaying a message detailing the collections feature and
+ * allowing users to easily start creating their first.
+ *
+ * @param showAddToCollectionButton Whether or not the "Add to Collection" button should be shown.
+ * @param onAddTabsToCollectionButtonClick Invoked when the user clicks on the "Add Tabs to Collection" button.
+ * @param onRemoveMessageClick Invoked when the user clicks on the close button to remove the Collections placeholder.
+ */
+@Composable
+fun CollectionsPlaceholder(
+    showAddToCollectionButton: Boolean,
+    onAddTabsToCollectionButtonClick: () -> Unit,
+    onRemoveMessageClick: () -> Unit,
+) {
+    Box(
+        modifier = Modifier
+            .semantics(mergeDescendants = true) {}
+            .dashedBorder(
+                color = FirefoxTheme.colors.borderPrimary,
+                cornerRadius = 8.dp,
+                dashHeight = 2.dp,
+                dashWidth = 4.dp
+            )
+            .padding(
+                start = 16.dp,
+                bottom = 16.dp
+            )
+    ) {
+        Column(
+            Modifier
+                .padding(all = 16.dp)
+                .fillMaxWidth()
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                SectionHeader(
+                    text = stringResource(R.string.collections_header),
+                    modifier = Modifier
+                        .weight(1f)
+                        .wrapContentHeight(align = Alignment.Top)
+                )
+
+                IconButton(
+                    onClick = onRemoveMessageClick
+                ) {
+                    Icon(
+                        painter = painterResource(R.drawable.ic_close),
+                        contentDescription = stringResource(
+                            R.string.remove_home_collection_placeholder_content_description
+                        ),
+                        modifier = Modifier.size(24.dp),
+                        tint = FirefoxTheme.colors.iconPrimary
+                    )
+                }
+            }
+
+            PrimaryText(
+                text = stringResource(R.string.no_collections_description2),
+                modifier = Modifier.fillMaxWidth(),
+                fontSize = 14.sp
+            )
+
+            if (showAddToCollectionButton) {
+                Spacer(modifier = Modifier.height(12.dp))
```

Comment:
This should be 8dp https://github.com/mozilla-mobile/fenix/blob/44fa89915467745af4ebf20fde1ab2eddac15411/app/src/main/res/layout/no_collections_message.xml#L61